### PR TITLE
Migrate `BeaconChainActions` to Chakra

### PR DESCRIPTION
**Pull request by soheil555**
## Description

Migrate the `BeaconChainActions` component to Chakra UI

## Related Issue
#8632

### CODE REVIEW

File: src/components/BeaconChainActions.tsx [hunk 1]
```diff
@@ -140,21 +108,30 @@ const BeaconChainActions: React.FC = () => {
...
-          </StyledButtonLink>
+          </ButtonLink>
           <ButtonLink variant="outline" to="/staking/">
             <Translation id="page-upgrades-index-staking-learn" />
           </ButtonLink>
-        </StyledCardLeft>
-        <StyledCardRight
+        </Box>
+        <Box
+          as={Card}
```
Comment (pettinarip) [inline src/components/BeaconChainActions.tsx:131]: `Card` is already exporting a chakra component. Lets refactor a bit the `Card` component to spread the receiving props:
```tsx
// in Card.tsx
const Card: React.FC<IProps> = ({...rest}) => (
  <Flex ... {...rest}>
```

```suggestion
        <Card
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Nice, gj @soheil555 thanks for the PR. Left one request.

File: src/components/BeaconChainActions.tsx (hunk 1 above)
Comment (soheil555) [reply src/components/BeaconChainActions.tsx]: Resolved

**Review by pettinarip — APPROVED** [review]
Awesome!

**Comment by pettinarip** [comment]
@soheil555 be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by pettinarip** [comment]
@all-contributors please add @soheil555 for code

**Pull request merged**